docs/using-controller-manager-with-kubeadm.md

```diff
@@ -28,20 +30,20 @@ gzip -d openstack-cloud-controller-manager.tgz
 docker load < openstack-cloud-controller-manager.tar
 ```
 
-Step 4: Create a configmap with the openstack cloud configuration (see `manifests/openstack-cloud-controller-manager/cloud-config`)
+Step 4: Create a configmap with the openstack cloud configuration (see `manifests/controller-manager/cloud-config`)
 ```
 kubectl create configmap cloud-config --from-file=/etc/kubernetes/cloud-config -n kube-system
 ```
 
 Step 5: Deploy controller manager
 
-Option #1 - Using a single pod with the definition in `manifests/openstack-cloud-controller-manager/openstack-cloud-controller-manager-pod.yaml`
+Option #1 - Using a single pod with the definition in `manifests/ontroller-manager/openstack-cloud-controller-manager-pod.yaml`
```

Comment:
`manifests/ontroller-manager` should be `manifests/controller-manager` (missing a 'c')

Reply:
Thanks, fixed in the latest commit.